confidant/services/jwkmanager.py

```diff
@@ -67,17 +69,20 @@ def get_jwt(self, kid: str, payload: dict,
             'exp': expiry,
         })
 
-        token = jwt.encode(
-            payload=payload,
-            headers={'kid': kid},
-            key=key.export_to_pem(private_key=True, password=None),
-            algorithm=algorithm,
-        )
+        with stats.timer('get_jwt.encode'):
+            # XXX: TODO: cache export_to_pem
```

Comment:
@erickduran , keeping track of this, we may be able to speed this function up by storing the pem in memory.